backend/src/middleware/handleHtmlContent/handleContentData.spec.js

```diff
@@ -359,25 +354,17 @@ describe('Hashtags', () => {
 
       describe('afterwards update the Post by removing a Hashtag, leaving a Hashtag and add a Hashtag', () => {
         // The already existing Hashtag has no class at this point.
-        const updatedPostContent =
+        const content =
           '<p>Hey Dude, <a class="hashtag" href="/search/hashtag/Elections">#Elections</a> should work equal for everybody!? That seems to be the only way to have equal <a href="/search/hashtag/Liberty">#Liberty</a> for everyone.</p>'
-        const updatePostMutation = gql`
-          mutation($postId: ID!, $postTitle: String!, $updatedPostContent: String!) {
-            UpdatePost(id: $postId, title: $postTitle, content: $updatedPostContent) {
-              id
-              title
-              content
-            }
-          }
-        `
 
         it('only one previous Hashtag and the new Hashtag exists', async () => {
           await mutate({
             mutation: updatePostMutation,
             variables: {
-              postId,
-              postTitle,
-              updatedPostContent,
+              id,
+              title,
+              content,
+              categoryIds,
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-20T14:21:24Z" title="Tuesday, August 20th 2019, 4:21:24 pm +02:00">Aug 20, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -359,25 +354,17 @@ describe('Hashtags', () => {
 
       describe('afterwards update the Post by removing a Hashtag, leaving a Hashtag and add a Hashtag', () => {
         // The already existing Hashtag has no class at this point.
-        const updatedPostContent =
+        const content =
           '<p>Hey Dude, <a class="hashtag" href="/search/hashtag/Elections">#Elections</a> should work equal for everybody!? That seems to be the only way to have equal <a href="/search/hashtag/Liberty">#Liberty</a> for everyone.</p>'
-        const updatePostMutation = gql`
-          mutation($postId: ID!, $postTitle: String!, $updatedPostContent: String!) {
-            UpdatePost(id: $postId, title: $postTitle, content: $updatedPostContent) {
-              id
-              title
-              content
-            }
-          }
-        `
 
         it('only one previous Hashtag and the new Hashtag exists', async () => {
           await mutate({
             mutation: updatePostMutation,
             variables: {
-              postId,
-              postTitle,
-              updatedPostContent,
+              id,
+              title,
+              content,
+              categoryIds,
```


@Tirokk, I changed the variable names for all these to reuse the mutation and variables and DRY it out a bit
I think it's clear that it's a post id, title, etc... since we are using the `CreatePost`/`UpdatePost` mutation/variables, but I could change this, if you think it's better.